4/main.js

```diff
@@ -0,0 +1,68 @@
+const sum = 0.1 + 0.3;
+console.log(sum.toFixed(1));
+const login = prompt("Введіть ваш логін.");
+const password = prompt("Введіть ваш пароль.");
+if (login == "admin") {
+  if (password == "12pass33210") {
+    console.log("Успішна авторизація!");
+  } else {
+    console.log("Пароль введено невірно.");
```

Comment:
We can't let anyone know what is wrong. Imaging scenario, while brurefocing. It is not safe to tell what exactly is wrong. So we are always saying - `Your login or password can be wrong`